packages/@jsii/go-runtime/jsii-experimental/runtime.go

```diff
@@ -8,6 +8,8 @@ import (
 	"regexp"
 )
 
+type implementationMap = map[reflect.Type]reflect.Type
```

Comment:
That could have used a documentation string explaining what the key versus value is supposed to be.